### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by HansOlsson**
Add new MCP as planned on last phone meeting.

### CODE REVIEW

File: RationaleMCP/ReadMe.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ New MCP should be added to the following list - on the main branch to keep track
 but the rest of the development on a branch/pull-request before being accepted.
 
 ## List of existing MCPs
-- MCP0035 Multilingual support od Modelica([MCP/0035](https://github.com/modelica/ModelicaSpecification/tree/MCP/0035/RationaleMCP/0035))
+- MCP0036 Setting statues ([MCP/0036]) (https://github.com/modelica/ModelicaSpecification/tree/MCP/0036/RationaleMCP/0036))
+- MCP0035 Multilingual support of Modelica([MCP/0035](https://github.com/modelica/ModelicaSpecification/tree/MCP/0035/RationaleMCP/0035))
```
Comment (henrikt-ma) [inline RationaleMCP/ReadMe.md:21]: (This spelling fix should go in a separate commit.)

File: RationaleMCP/ReadMe.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ New MCP should be added to the following list - on the main branch to keep track
 but the rest of the development on a branch/pull-request before being accepted.
 
 ## List of existing MCPs
-- MCP0035 Multilingual support od Modelica([MCP/0035](https://github.com/modelica/ModelicaSpecification/tree/MCP/0035/RationaleMCP/0035))
+- MCP0036 Setting statues ([MCP/0036]) (https://github.com/modelica/ModelicaSpecification/tree/MCP/0036/RationaleMCP/0036))
```
Comment (henrikt-ma) [inline RationaleMCP/ReadMe.md:20]: Spelling mistake in _statues_.

**Comment by gkurzbach** [comment]
Sorry, but what was the concrete aim of this MCP?
